### ISSUE TITLE
Requires GTK+ 3.0 installed on Fedora 24. How to install it?

### ISSUE THREAD

**Issue by vodolaz095** [andlabs/ui#216]
```
[vodolaz095@steel ui]$ go test
# pkg-config --cflags gtk+-3.0
Package gtk+-3.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `gtk+-3.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'gtk+-3.0' found
pkg-config: exit status 1
FAIL	github.com/andlabs/ui [build failed]
```

**Comment by vodolaz095** [comment]
possible fix (fedora 24) is 

```shell

  # dnf install gtk3 gtk3-devel

```

**Comment by andlabs** [comment]
(just to note that this is not a bug but rather a prerequisite — GTK+ is required to use package ui; I don't know the correct Fedora commands, sorry)

**Comment by sndnvaps** [comment]
dnf search gtk+-3.0
it that possible

**Comment by andlabs** [comment]
`sudo dnf install gtk3-devel`

Closing as duplicate of resolved #255.
